pipelines/resources/contrib/contrib-gpu-operator.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: contrib-gpu-operator
+  annotations:
+    description: |
+      This task will deploy GPU/NFD on the Spoke Cluster.
+spec:
+  workspaces:
+    - name: ztp
+  params:
+    - name: ztp-container-image
+      type: string
+      default: "quay.io/ztpfw/pipeline:latest"
+    - name: kubeconfig
+      type: string
+      default: ""
+    - name: spokes-config
+      type: string
+      default: ""
+    - name: mock
+      type: string
+      default: "false"
+  stepTemplate:
+    env:
```

Comment:
Should we add a parameter to set or unset the deployment of the GPU Operator (at Tekton lvl)?